config/samples/zookeeper_sample.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: zookeeper.pravega.io/v1beta1
```

Comment:
We should not add zookeeperCluster CR into koperator sample files.
I suggest to use template files for tests like here: https://github.com/banzaicloud/koperator/blob/8ee5d6dad71b558585a780e77f1aa824f64d2d5b/tests/e2e/kafka_prod_consum_test.go#L44